pkg/cluster/spec/cdc.go

```diff
@@ -152,8 +154,10 @@ func (i *CDCInstance) InitConfig(
 	instanceConfig := spec.Config
 
 	configFileSupported := false
+	dataDirSupported := false
 	if semver.Compare(clusterVersion, "v4.0.13") >= 0 && clusterVersion != "v5.0.0-rc" {
 		configFileSupported = true
+		dataDirSupported = true
```

Comment:
better use a dedicated if block, so that the error message can be specific to `data_dir`.

Reply:
and did you make sure v5.0 series versions except `v5.0.0-rc` support the `--sort-dir` argument?

Reply:
yes, it's checked at source code level.

Reply:
yes, it's checked at the source code level.

Reply:
> better use a dedicated if block, so that the error message can be specific to `data_dir`.

I don't think we have to throw an error message on `data-dir`, the reason is described above.